Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
             } else {
+
                 jQuery(filters_container).slideUp();
+
             }
+
         });
 
-        jQuery('.sonata-filter-form', subject).on('submit', function () {
-            jQuery(this).find('[sonata-filter="true"]:hidden :input').val('');
+        jQuery(".sonata-filter-form", subject).on("submit", function () {
+
+            jQuery(this).find("[sonata-filter=\"true\"]:hidden :input").
+                val("");
+
         });
 
         /* Advanced filters */
-        if (jQuery('.advanced-filter :input:visible', subject).filter(function () { return jQuery(this).val() }).length === 0) {
-            jQuery('.advanced-filter').hide();
-        };
+        if (jQuery(".advanced-filter :input:visible", subject).filter(function () {
+
+            return jQuery(this).val();
+
+        }).length === 0) {
+
+            jQuery(".advanced-filter").hide();
+
+        }
+
+        jQuery("[data-toggle=\"advanced-filter\"]", subject).click(() => {
+
+            jQuery(".advanced-filter").toggle();
 
-        jQuery('[data-toggle="advanced-filter"]', subject).click(function() {
-            jQuery('.advanced-filter').toggle();
         });
+
     },
 
     /**
      * Change object field value
      * @param subject
      */
-    set_object_field_value: function(subject) {
-        Admin.log('[core|set_object_field_value] set value field on', subject);
+    "set_object_field_value" (subject) {
+
+        Admin.log("[core|set_object_field_value] set value field on", subject);
+
+        this.log(jQuery("a.sonata-ba-edit-inline", subject));
+        jQuery("a.sonata-ba-edit-inline", subject).click(function (event) {
 
-        this.log(jQuery('a.sonata-ba-edit-inline', subject));
-        jQuery('a.sonata-ba-edit-inline', subject).click(function(event) {
             Admin.stopEvent(event);
-            var subject = jQuery(this);
+            const subject = jQuery(this);
             jQuery.ajax({
-                url: subject.attr('href'),
-                type: 'POST',
-                success: function(response) {
-                    var elm = jQuery(subject).parent();
+                "url": subject.attr("href"),
+                "type": "POST",
+                "success" (response) {
+
+                    const elm = jQuery(subject).parent();
                     elm.children().remove();
-                    // fix issue with html comment ...
+                    // Fix issue with html comment ...
                     elm.html(jQuery(response.replace(/<!--[\s\S]*?-->/g, "")).html());
-                    elm.effect("highlight", {'color' : '#57A957'}, 2000);
+                    elm.effect("highlight", {"color": "#57A957"}, 2000);
                     Admin.set_object_field_value(elm);
+
                 },
-                error: function(xhr, statusText, errorThrown) {
-                    jQuery(subject).parent().effect("highlight", {'color' : '#C43C35'}, 2000);
+                "error" (xhr, statusText, errorThrown) {
```

Comment:
- eslint: Expected object keys to be in ascending order. 'error' should be before 'success'.
- eslint: 'errorThrown' is defined but never used.